src/rubrix/client/models.py

```diff
@@ -246,6 +248,13 @@ class TokenClassificationRecord(_Validators):
     metrics: Optional[Dict[str, Any]] = None
     search_keywords: Optional[List[str]] = None
 
+    __chars2tokens__: Dict[int, int] = None
+    __tokens2chars__: Dict[int, Tuple[int, int]] = None
+
+    def __init__(self, **data):
+        super().__init__(**data)
+        self.__chars2tokens__, self.__tokens2chars__ = self.__build_indices_map__()
```

Comment:
@dcfidalgo  better if we defined both maps as lazy properties. Any idea about how to do that?